features/step_definitions/scheduler.rb

```diff
@@ -24,5 +24,15 @@
     opts = {resource: 'scheduler', resource_name: name, p: patch_json, type: 'merge' }
     @result = _admin.cli_exec(:patch, **opts)
     raise "Cannot restore scheduler: #{name}" unless @result[:success]
+    conditions= cluster_operator("kube-scheduler").conditions
+    expected = {"Degraded"=>"False", "Progressing"=>"False", "Available"=>"True"}
+    sleep 300
```

Comment:
@kasturinarra please don't use hardcoded sleep

Reply:
@pruan-rht agree, could you please  give me some pointers on how to come across this ?

Reply:
@kasturinarra do you mean how to avoid the hard sleep?  Please refer to https://github.com/openshift/verification-tests/blob/master/lib/openshift/pod.rb#L268 for an example.  Basically you use the `wait_for` call which will exit as soon as the condition check is meet.

Reply:
@pruan-rht i tried using wait_for with the below steps [1] but i always end up seeing premature failures, so can we keep it this way for now ?

[1] http://pastebin.test.redhat.com/879302

Reply:
@pruan-rht could you please help review again ? Thanks !!